test/src/receiver/main.cpp

```diff
@@ -141,20 +188,25 @@ int main() {
     };
     std::shared_ptr<cadmium::dynamic::modeling::coupled<TIME>> TOP =
         std::make_shared<cadmium::dynamic::modeling::coupled<TIME>>(
-                                                                    "TOP", 
-                                                                    submodels_TOP, 
+                                                                    "TOP",
+                                                                    submodels_TOP,
                                                                     iports_TOP,
-                                                                    oports_TOP, 
-                                                                    eics_TOP, 
-                                                                    eocs_TOP, 
+                                                                    oports_TOP,
+                                                                    eics_TOP,
+                                                                    eocs_TOP,
                                                                     ics_TOP);
 
-    ///****************////
+    /**
+     * In this model, runner are created and also the time to create
+     * them are measured. Once they are created simulation starts and
+     * the time it took to complete the simulation and simulation is ran until

```

Comment:
grammatical error can be corrected. It would 'simulation is run until..'

Reply:
solved